README.md

```diff
@@ -93,37 +129,19 @@ Run `foundry version` to verify it's installed correctly.
-* Work with the Foundry capabilities of this app
-    * US-1: [App capabilities](https://falcon.crowdstrike.com/documentation/category/u0daabab/app-capabilities)
-    * US-2: [App capabilities](https://falcon.us-2.crowdstrike.com/documentation/category/u0daabab/app-capabilities)
-    * EU-1: [App capabilities](https://falcon.eu-1.crowdstrike.com/documentation/category/u0daabab/app-capabilities)
+## Running, deploying and installing Custom Apps
+
+For detailed info about running, deploying and installing custom apps in your CID, see the Falcon Foundry product documentation:
+
+* Overview and setup: [US-1](https://falcon.crowdstrike.com/documentation/page/f5f7cd69/falcon-console-user-interface-capabilities), [US-2](https://falcon.us-2.crowdstrike.com/documentation/page/f5f7cd69/falcon-console-user-interface-capabilities), [EU](https://falcon.eu-1.crowdstrike.com/documentation/page/f5f7cd69/falcon-console-user-interface-capabilities)
+* Deploy an app: [US-1](https://falcon.crowdstrike.com/documentation/page/ofd46a1c/deploy-an-app), [US-2](https://falcon.us-2.crowdstrike.com/documentation/page/ofd46a1c/deploy-an-app), [EU](https://falcon.eu-1.crowdstrike.com/documentation/page/ofd46a1c/deploy-an-app)
+* Create a new app using this app as template: [US-1](https://falcon.crowdstrike.com/documentation/page/l159717b/create-an-app#c4378b86), [US-2](https://falcon.us-2.crowdstrike.com/documentation/page/l159717b/create-an-app#c4378b86), [EU](https://falcon.eu-1.crowdstrike.com/documentation/page/l159717b/create-an-app#c4378b86)
+* Run an app in development mode after deployment: [US-1](https://falcon.crowdstrike.com/documentation/page/fb88e442/view-and-manage-apps#d5175ae2), [US-2](https://falcon.us-2.crowdstrike.com/documentation/page/fb88e442/view-and-manage-apps#d5175ae2), [EU](https://falcon.eu-1.crowdstrike.com/documentation/page/fb88e442/view-and-manage-apps#d5175ae2)
+* Work with the Foundry capabilities of an app: [US-1](https://falcon.crowdstrike.com/documentation/category/u0daabab/app-capabilities), [US-2](https://falcon.us-2.crowdstrike.com/documentation/category/u0daabab/app-capabilities), [EU](https://falcon.eu-1.crowdstrike.com/documentation/category/u0daabab/app-capabilities)
 
 ## Foundry resources
 
-See our product documentation:
-* US-1: [Falcon Foundry](https://falcon.crowdstrike.com/documentation/category/c3d64B8e/falcon-foundry)
-* US-2: [Falcon Foundry](https://falcon.us-2.crowdstrike.com/documentation/category/c3d64B8e/falcon-foundry)
-* EU-1: [Falcon Foundry](https://falcon.eu-1.crowdstrike.com/documentation/category/c3d64B8e/falcon-foundry)
+See our product documentation: [US-1](https://falcon.crowdstrike.com/documentation/category/c3d64B8e/falcon-foundry), [US-2](https://falcon.us-2.crowdstrike.com/documentation/category/c3d64B8e/falcon-foundry), [EU1](https://falcon.eu-1.crowdstrike.com/documentation/category/c3d64B8e/falcon-foundry)
```

Comment:
Add a link to the Foundry Learn page too?

Reply:
That's a good idea. Eventually, we might want to link to the Tech Hub and filter by Foundry too. 

Should I add a line similar to the one above that says "See our learning resources: US-1, US-2, EU"? or would it be better to inline it? 

> See our product documentation (US-1, US-2, EU) and Foundry learning resources (US-1, US-2, EU) for more information.

Reply:
Our style generally prefers bulleted lists so I'd suggest 2 bullets right under the section header with no intro text, in the same style as the release emails:

- Foundry documentation US-1 | US-2 | EU
- Foundry learning resources US-1 | US-2 | EU

and then you'll be able to add more resources whenever they're ready, with no rewording necessary.

Reply:
I haven't seen a release email. Is it possible to forward an example or link to one? I'm curious to know if there's a colon before the regions with links.

Reply:
Forwarded! Somewhat oddly, there isn't a colon. But adding one would be OK :-)

Reply:
Done!